packages/shared/src/entity.types.ts

```diff
@@ -1,6 +1,6 @@
 import moment from 'moment-timezone';
 
-export type ColumnValue = string | boolean | number | null;
+export type ColumnValue = object | string | boolean | number | null;
```

Comment:
In what case is `ColumnValue` an `object`?  I am guessing this is here due to `Category` having a `Keyword` still, but it would be great if we could avoid it, since it could be a bit too open.

Reply:
I had to use this as a hack while you were dealing with the nested JSON in custom columns. Glad you caught this though, I removed it.